pom.xml

```diff
@@ -169,7 +170,7 @@
 		<mockito.version>3.11.1</mockito.version>
 		<objenesis.version>3.2</objenesis.version>
 		<openjson.version>1.0.12</openjson.version>
-		<slf4j.version>1.7.25</slf4j.version>
+		<slf4j.version>2.0.0-alpha1</slf4j.version>
```

Comment:
Why to update to `alpha`?

Reply:
Because SLF4J added support for JPMS in 2.x.
They just released alpha2. 
Hopefully they will release 2.0.0 soon! But SLF4J project is not very active lately.

Reply:
they had alphas and betas only for very long time :(
let's see :))